runtime/src/main/scala/akka/grpc/internal/AkkaGrpcClientFactory.scala

```diff
@@ -6,19 +6,21 @@ package akka.grpc.internal
 
 import scala.concurrent.ExecutionContext
 import scala.reflect.{ ClassTag, classTag }
-
 import akka.grpc.GrpcClientSettings
 import akka.grpc.scaladsl.AkkaGrpcClient
 import akka.stream.Materializer
+import language.reflectiveCalls
 
 object AkkaGrpcClientFactory {
   def create[T <: AkkaGrpcClient: ClassTag](settings: GrpcClientSettings)(implicit mat: Materializer, ex: ExecutionContext): T = {
-    classTag[T].runtimeClass.asInstanceOf[Class[T]]
-      .getConstructor(
-        classOf[GrpcClientSettings],
-        classOf[Materializer],
-        classOf[ExecutionContext])
-      .newInstance(settings, mat, ex)
+    // this reflection requires:
```

Comment:
This is no longer true... perhaps remove comment? when someone is this deep in the internals already, I guess it's not such a big step to get it from the code anymore? Plus this is expected to change again later.